.github/workflows/ci.yml

```diff
@@ -380,13 +380,15 @@ jobs:
         -DCMAKE_PREFIX_PATH:PATH=$(pwd)/../dependencies/install/
         -DF3D_COVERAGE=ON
         -DF3D_MODULE_EXTERNAL_RENDERING=ON
+        -DF3D_MODULE_EXR=ON
         -DF3D_MODULE_RAYTRACING=ON
         -DF3D_PLUGINS_STATIC_BUILD=OFF
         -DF3D_PLUGIN_BUILD_ALEMBIC=ON
         -DF3D_PLUGIN_BUILD_ASSIMP=ON
         -DF3D_PLUGIN_BUILD_DRACO=ON
         -DF3D_PLUGIN_BUILD_OCCT=ON
         -DF3D_STRICT_BUILD=ON
+        -DF3D_EXCLUDE_DEPRECATED=ON
```

Comment:
should be off then ?

Reply:
It's only the coverage CI. The main CI has the default value (OFF)